### PR TITLE
use the new run-build command

### PR DESCRIPTION
**Pull request by csrwng**
switches the new pipeline build job in the library to use the run-build command in release-ci

### CODE REVIEW

**Comment by csrwng** [comment]
@stevekuznetsov should be ready for review

File: vars/launchBuild.groovy [hunk 1]
```diff
@@ -29,9 +29,7 @@ def call(ctx, jobName, jobId, buildName, toTag, dockerfile) {
       ],
       "spec": buildSpec
     ]
-    ctx.runAndWaitForBuild(ctx,
-      build.metadata.name,
-      { return ctx.openshift.apply(build) }
-    )
+    def buildJSON = groovy.json.JsonOutput.prettyPrint(groovy.json.JsonOutput.toJson(build))
```
Comment (stevekuznetsov) [inline vars/launchBuild.groovy:32]: Why do we need to `prettyPrint`?

Comment (csrwng) [reply vars/launchBuild.groovy:32]: Because it gets output in the Jenkins job log. It’s much easier to read.

**Pull request merged**